### DISCUSSION TITLE
Automation Manager - conditional scripts

### DISCUSSION THREAD

**Discussion by WouterLS** [amidaware/tacticalrmm#1106] [Ideas]
Hi,

Would it be possible to make it so you can have a script check based on some condition.
For instance it would be nice to be able to some script check on machines that have a specific piece of software installed

**Comment by dinger1986** [comment]
Can you not write that into your script?

If x folder exists do whatever?

**Reply by WouterLS** [reply]
Yes, you could.. 

That would mean all agents still have to download & execute the script.
With some kind of conditional script check deployment it would be alot cleaner. 
the interface under checks would also have less. Which gives it a better look.

Just an idea
